pages/evaluation.md

```diff
@@ -1,35 +1,47 @@
-- Hybrids of these two subspecies.
-- The aim of this test set is to see whether an anomaly detection algorithm for butterflies is transferrable across mimic butterflies.
+The goal of the challenge is to elicit new ways of solving this problem through designing a model to distinguish between non-hybrid instances and hybrid instances.
 
-There are 2 phases:
+## The Second Component of the Test Dataset
+Moreover, Species A has a [mimic](https://en.wikipedia.org/wiki/M%C3%BCllerian_mimicry) Species B. These two butterfly species have quite similar appearances to evade predators. Specifically, for the two subspecies of Species A that produce the signal hybrid, there are mimic subspecies of Species B which also hybridize. We thus provide a separate, second test data set that contains:
+- An image collection of butterfly **Species B subspecies**. These are non-hybrid instances.
+- An image collection of **Species B hybrids**, which are from parents from different subspecies of Species B. 
+
+Specifically, we only consider one particular hybrid and two specific parent subspecies of Species B, corresponding to Species A signal hybrid and its two parent subspecies. 
+
+This second test set aims to investigate whether an anomaly detection algorithm for butterfly Species A is transferrable to the mimic butterfly Species B.
+
+## Evaluation Phases
+There are 2 phases. Each test data set is split into a development set and a final test set.  
 1. **Development phase:**
-	* The provided training data contains 1) images of all Species A's subspecies: these are considered normal (not anomaly) cases; 2) a signal set comprising the "signal" hybrid (anomaly) cases: the most common hybrids of Species A's subspecies in _this_ dataset.
-	* The goal is to develop an algorithm to 1) detect hybrid butterflies of Species A's subspecies among their parent subspecies, and 2) detect the "signal" hybrid butterfly of a mimic Species (Species B) among its two parent subspecies.
-	* Upload your model; feedback will be provided on the development set for each component until the end of the challenge:
-		1. Signal and non-signal hybrid subspecies of Species A. 
-		2. Signal hybrid subspecies of a mimic Species B.
-	* Participants may submit _one_ score on the "development" sets to be displayed on the leaderboard. This score can be removed and replaced with a newer or better score as they choose.
+	* The provided training data contains:
+		- Images of all Species A subspecies: these images are considered "normal" (not anomaly) cases.
+		- A signal set comprising the most common hybrid: these images are considered anomaly cases.*
+	* The goal is to develop an algorithm to detect hybrid instances (the anomaly cases).
+	* Upload your model: feedback will be provided on the development set until the end of the challenge:
+		1. Detect signal and non-signal hybrid subspecies of Species A. 
+		2. Detect signal hybrid subspecies of the mimic Species B.
```

Comment:
```suggestion
		2. Detect hybrid subspecies among the mimic Species B (Species B subspecies are mimics of the Species A signal hybrid parent subspecies).
```
Attempt at clarifying. @pujols, @hlapp

Reply:
I would change _hybrid subspecies_ to _hybrids of subspecies_ or _subspecies hybrids_. (The subspecies themselves are normally not hybrids.)